src/java.desktop/macosx/classes/sun/awt/CGraphicsEnvironment.java

```diff
@@ -139,12 +143,14 @@ void _displayReconfiguration(int displayId, boolean removed) {
         rebuildDevices();
     }
 
-    @Override
-    @SuppressWarnings("removal")
-    protected void finalize() throws Throwable {
-        try {
-            super.finalize();
-        } finally {
+    private static class CGEDisposerRecord implements DisposerRecord {
+        private long displayReconfigContext;
```

Comment:
`displayReconfigContext` should be final

Reply:
ok